### PR TITLE
Add the next clickhouse versions we are targeting for upgrades

### PR DESCRIPTION
**Pull request by lynnagara**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -120,6 +120,22 @@ def update(self) -> Image:
             'sha256:125d2ea49c298515c46784d202a2bd4dde05157c85a76517afc2567f262ab335',  # noqa: E501
         ),
     ),
+    Image(
+        registry='registry-1.docker.io',
+        source='altinity/clickhouse-server',
+        tag='22.3.15.34.altinitystable',
+        digests=(
+            'sha256:5a67ec149acc13e3d87ed1e3b94b4ada6f0acdc75145724959bbd8c0a6f18410',  # noqa: E501
```
Comment (asottile-sentry) [inline main.py:129]: hmm these have arm images so I expect them to have two digests

Comment (asottile-sentry) [reply main.py:129]: oh can you add `arm64/v8` to the ARCHS sequence and re-import these? it should pull in the other digests then

Comment (lynnagara) [reply main.py:129]: I think i created this wrong. Or at least I did it manually - is there automation for this?

Comment (asottile-sentry) [reply main.py:129]: yeah the script has a `sync` / `update` / etc. subcomand 👍

**Review by asottile-sentry — APPROVED** [review]

**Pull request merged**